src/color-picker/color-picker.scss

```
@@ -0,0 +1,144 @@
+  @include iui-dropdown;
+  width: $iui-baseline * 27;
+  border-radius: $iui-border-radius;
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: $iui-sm;
+  padding-top: $iui-sm;
+
+  &.iui-advanced {
+    width: $iui-baseline * 24;
+  }
+}
+
+@mixin iui-color-option {
+  @include iui-focus;
+  height: $iui-l;
+  width: $iui-l;
+  margin-right: $iui-sm;
+  margin-bottom: $iui-sm;
+  cursor: pointer;
+  border-radius: $iui-border-radius;
+  background-color: var(--option-color);
+
+  @include themed {
+    box-shadow: inset 0 0 0 $iui-xxs
+      rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
+  }
+
+  &.iui-active {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs t(iui-color-foreground-primary);
+    }
+  }
+
+  &:hover {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+    }
+  }
+
+  &.iui-add-color-button {
+    box-shadow: none;
+  }
+}
+
+@mixin iui-color-palette {
+  width: auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+@mixin iui-saved-colors {
+  margin-left: $iui-sm;
+}
+
+@mixin iui-color-selector {
+  padding-left: $iui-sm;
+  padding-top: $iui-sm;
+  display: flex;
+  flex-wrap: wrap;
+  height: $iui-baseline * 16;
+
+}
+
+@mixin iui-color-field {
+  position: relative;
+  cursor: crosshair;
+  width: $iui-baseline * 19;
+  height: auto;
+
+  @include themed {
+    background-image: linear-gradient(0deg, rgb(0, 0, 0), transparent),
+      linear-gradient(90deg, rgb(255, 255, 255), var(--color));
+  }
+}
+
+@mixin iui-color-slider-wrapper {
+  margin-left: $iui-sm;
+  position: relative;
+  width: $iui-s;
+  height: auto;
+}
+
+@mixin iui-color-slider {
+  inset: 0;
+  position: absolute;
+  border-radius: $iui-border-radius;
+
+  @include themed {
+    background: linear-gradient(
+      $iui-color-system-red 0%,
+      $iui-color-system-yellow 17%,
+      rgb(0, 255, 0) 33%,
+      rgb(0, 255, 255) 50%,
+      $iui-color-system-blue 67%,
+      $iui-color-system-magenta 83%,
+      $iui-color-system-red 100%
+    );
+  }
+}
+
+@mixin iui-color-selection-wrapper {
+  position: absolute;
+  top: var(--top, 0%);
+  left: var(--left, 0%);
+}
+
+@mixin iui-color-selection {
+  @include iui-focus;
+  width: $iui-s + 2;
+  height: $iui-s + 2;
+  border-radius: 50%;
+  transform: translate(-3px, -3px);
+  @include themed {
+    box-shadow: rgba(t(iui-color-foreground-accessory-rgb), t(iui-opacity-1)) 0
+      0 0 $iui-xxs - 1,
+      inset 0 0 0 $iui-xxs - 1
+      rgba(t(iui-color-foreground-accessory-rgb), t(iui-opacity-6));
+    background-color: var(--selected-color);
+  }
+
+  &.iui-white {
+    @include iui-focus;
+    transform: translate(-1px, -9px);
+    @include themed {
+      box-shadow: rgba(0, 0, 0, 0.37) 0 1px 4px 0;
+      background-color: #FFF;
+    }
+  }
+
+  &:hover {
+    @include themed {
+      box-shadow: 0 0 0 $iui-xxs
+        rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+    }
+  }
+} ;
```

Please run prettier before pushing to avoid these formatting blunders. If it doesn't fix it, then change it manually
```suggestion
}
```